prime-router/metadata/schemas/Experity/experity-covid-19.schema

```diff
@@ -0,0 +1,644 @@
+
+    - name: ordering_facility_street
+      csvFields: [{name: Ordering_facility_street}]
+
+    - name: ordering_facility_city
+      csvFields: [{name: Ordering_facility_city}]
+
+    - name: ordering_facility_state
+      csvFields: [{name: Ordering_facility_state}]
+
+    - name: ordering_facility_zip_code
+      csvFields: [{name: Ordering_facility_zip_code}]
+
+    - name: ordering_facility_phone_number
+      csvFields: [{name: Ordering_facility_phone_number}]
+
+#testing lab
+    - name: testing_lab_name
+      mapper: concat(testing_lab_name_temp, ordering_facility_name_temp)
+
+    - name: testing_lab_street
+      csvFields: [{name: Testing_lab_street}]
+
+    - name: testing_lab_city
+      csvFields: [{name: Testing_lab_city}]
+
+    - name: testing_lab_state
+      csvFields: [{name: Testing_lab_state}]
+
+    - name: testing_lab_zip_code
+      csvFields: [{name: Testing_lab_zip_code}]
+
+    - name: testing_lab_phone_number
+      csvFields: [{name: Ordering_facility_phone_number}]
+
+    - name: patient_id_assigner
+      csvFields: [{name: Ordering_facility_name}]
+
+#CLIA
+    - name: testing_lab_clia
+      csvFields: [{name: Testing_lab_ID}]
+
+    - name: testing_lab_id
+      csvFields: [{name: Testing_lab_ID}]
+
+    - name: reporting_facility_clia
+      csvFields: [{name: Testing_lab_ID}]
+
+    - name: filler_clia
+      csvFields: [{name: Testing_lab_ID}]
+
+
+#AOE fields
+    - name: pregnant
+      type: CODE
+      valueSet: sender-automation/pregnant
+      documentation: Translate multiple inbound values into the Pregnancy SNOMED Codes
+      csvFields: [{ name: Pregnant, format: $display}]
+      default: ""
+
+    - name: first_test
+      type: CODE
+      valueSet: sender-automation/yesno
+      documentation: Translate multiple inbound Y/N/U AOE values to RS values
+      csvFields: [{ name: First_test, format: $display}]
+      default: ""
+
+    - name: employed_in_healthcare
+      type: CODE
+      valueSet: sender-automation/yesno
+      documentation: Translate multiple inbound Y/N/U AOE values to RS values
+      csvFields: [{ name: Employed_in_healthcare, format: $display}]
+      default: ""
+
+    - name: symptomatic_for_disease
+      type: CODE
+      valueSet: sender-automation/yesno
+      documentation: Translate multiple inbound Y/N/U AOE values to RS values
+      csvFields: [{ name: Symptomatic_for_disease, format: $display}]
+      default: ""
+
+    - name: illness_onset_date
+      default: ""
+      csvFields: [{name: Illness_onset_date, format: "M/d/yyyy"}]
+
+
+# Default Values
+    - name: hospitalized
+      default: N
+
+    - name: icu
+      default: N
+
+
+#this lookup is in the covid-19 schema, but doesn't appear to be done in practice (?)
+    - name: equipment_model_name
+      table: LIVD-SARS-CoV-2-2021-04-28
+      tableColumn: Model
+      mapper: livdLookup()
+
+
+#unused fields - these are added here to suppress warnings.
```

Comment:
ok, well this is a hack for sure.   We need a better way to suppress warnings.   I created ticket #2022  - could you add any useful comments there?   thanks

Reply:
I'll add some comments.